state/state.go

```diff
@@ -690,7 +690,7 @@ func IsUpgradeInProgressError(err error) bool {
 // cannot be higher than the controller version.
 func (st *State) SetModelAgentVersion(newVersion version.Number, ignoreAgentVersions bool) (err error) {
 	if newVersion.Compare(jujuversion.Current) > 0 && !st.IsController() {
-		return errors.Errorf("a hosted model cannot have a higher version than the server model: %s > %s",
+		return errors.Errorf("hosted models cannot be a later version than the controller: %s > %s: upgrade 'controller' model first",
```

Comment:
hosted models cannot be upgraded to a later version that the controller
?
I personally like "newer version than the controller", but I don't know that its better.

But I do like "cannot be upgraded to" vs "cannot be".

I'm flexible, though.